controller/users.go

```diff
@@ -165,29 +154,6 @@ func (c *UsersController) checkPreviewUser(email string) (bool, error) {
 	return regexp.MatchString(c.config.GetIgnoreEmailInProd(), strings.ToLower(email))
 }
 
-func (c *UsersController) linkUserToRHD(ctx *app.CreateUsersContext, identityID string, rhdUsername string, rhdUserID string, protectedAccessToken string) error {
```

Comment:
You are not changing this code but just to avoid conflicts (if we create a new PR) we can address this in this PR too:
lines 124-127:
```
 	// If it's a new user, Auth service generates an Identity ID for the user.
 	identityID, err := uuid.FromString(uuid.NewV4().String())
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, err))
 	}
```
Should be just:
```
 	// If it's a new user, Auth service generates an Identity ID for the user.
 	identityID := uuid.NewV4()
```

Reply:
done in 5e17b5a